spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/BigQueryStrategy.scala

```diff
@@ -101,6 +101,20 @@ class BigQueryStrategy(expressionConverter: SparkExpressionConverter, expression
     Some(sourceQuery.bigQueryRDDFactory)
   }
 
+  /** Removes project plan with no fields and returns the child, to continue the generation of query.
+   * Spark is generating an empty project plan when the query has count(*)
+   * Found this issue when executing query-9 of TPC-DS
+   *
+   * @param plan
```

Comment:
please document the parameter and return value

Reply:
Added